runWorkflowXToYggHbb.py

```diff
@@ -178,8 +180,8 @@ def getLimit(masses, config, mggl, mggh, procTemplate):
             'mkdir -p Outputs/CollectedPlots_'+procTemplate+'/Combine/Results; ' + \
             'cp Combine/*combine_results_'+procTemplate+'_* Outputs/CollectedPlots_'+procTemplate+'/Combine/Results; ' + \
             'cp -r Combine/Models Outputs/CollectedPlots_'+procTemplate+'/Combine/Models; ' + \
-            'mkdir -p Outputs/CollectedPlots_'+procTemplate+'/Combine/Impacts; ' + \
-            'cp Combine/impacts* Outputs/CollectedPlots_'+procTemplate+'/Combine/Impacts/; ' + \
+#            'mkdir -p Outputs/CollectedPlots_'+procTemplate+'/Combine/Impacts; ' + \
+#            'cp Combine/impacts* Outputs/CollectedPlots_'+procTemplate+'/Combine/Impacts/; ' + \
```

Comment:
I understand that you move this to its dedicated function, which makes sense. Could you remove these lines though?

Reply:
changed.